sandbox/src/components/ContentSecurityPolicy.js

```diff
@@ -20,7 +20,7 @@ export default function ContentSecurityPolicy() {
         http-equiv="Content-Security-Policy"
         // cdn.tt.omtrdc.net is necessary for Target VEC to function properly.
         // *.sc.omtrdc.net is necessary for Analytics Data Insertion API to function properly
-        content={`default-src 'self';
+        content={`default-src 'self' blob:;
```

Comment:
What does `blob:` do here

Reply:
To support Blob URLs as intended in our code
https://github.com/adobe/alloy/blob/e142bdc9747e86a54cd9fefaef23b31aa257d079/src/components/Personalization/in-app-message-actions/actions/displayIframeContent.js#L142